integration_test/circt-lec/hw.mlir

```diff
@@ -40,3 +40,18 @@ hw.module @notnot(%in: i1) -> (out: i1) {
 // hw.output
 //  RUN: circt-lec %s -c1=basic -c2=basic -v=false | FileCheck %s --check-prefix=HW_OUTPUT
 //  HW_OUTPUT: c1 == c2
+
+
+hw.module @constZeroZero(%in: i1) -> (o1: i1, o2: i1) {
+  %zero = hw.constant 0 : i1
+  hw.output %zero, %zero : i1, i1
+}
+
+hw.module @xorZeroZero(%in: i1) -> (o1: i1, o2: i1) {
+  %zero = comb.xor bin %in, %in : i1
+  hw.output %zero, %zero : i1, i1
+}
+
+// hw.twooutputs
```

Comment:
The `hw.` comments that you see in this file are specifically labelling the tests with the operations in the `hw` dialect that they are testing - since this isn't testing an operation called `hw.twooutputs`, it probably makes sense to just replace this with a comment explaining what is being tested

Reply:
Yeah, that does make a lot more sense. I wasn't paying attention there.